src/test/rustdoc-gui/anchors.goml

```diff
@@ -11,7 +11,7 @@ reload:
 
 assert-css: ("#toggle-all-docs", {"color": "rgb(0, 0, 0)"})
 assert-css: (".fqn .in-band a:nth-of-type(1)", {"color": "rgb(0, 0, 0)"})
-assert-css: (".fqn .in-band a:nth-of-type(2)", {"color": "rgb(173, 68, 142)"})
+assert-css: (".fqn .in-band a:nth-of-type(2)", {"color": "rgb(173, 55, 138)"})
```

Comment:
Yikes, it's worrying that only one test had to change because of this massive color change ;)

Reply:
I think more tests will be added. :)